src/starbunk/bots/reply-loader.ts

```
@@ -0,0 +1,130 @@
+	private static logLoadingSummary(loadedBots: ReplyBot[]): void {
+		logger.info(`📊 Successfully loaded ${loadedBots.length} reply bots`);
+		if (loadedBots.length > 0) {
+			logger.info('📋 Reply bots summary:');
+			loadedBots.forEach((bot) => {
+				logger.info(`   - ${bot.defaultBotName} (${bot.constructor.name})`);
+			});
+		}
+	}
+}
+
+// Bind the static method to its class to preserve 'this' context
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)